spec/features/incoming_mail_handler_spec.rb

```diff
@@ -53,6 +55,12 @@
     expect(resp.action).to eq(IncomingMail::Response::COMMENT)
     expect(resp.comment.proposal.existing_observation_for(user)).to be_truthy
     expect(resp.comment.proposal.existing_approval_for(user)).to be_falsey
+    expect(my_approval.user).to_not eq(my_proposal.individual_approvals.last.user)
+    expect(deliveries.length).to eq(4)
+    expect(deliveries.select{|m| m.to.first == user.email_address}.length).to eq(1)
```

Comment:
nice update! In the future I might wrap something like this so it reads like:

`expect_one_email_sent_to(user.email_address)`


```ruby
def expect_one_email_sent_to(email_address)
  expect(deliveries.select { |email| email.to.first == email_address }.length).to eq (1)
end
```